test/utils/describeConformance.tsx

```diff
@@ -41,6 +41,7 @@ export interface InputConformanceOptions {
     mount: (node: React.ReactNode) => ReactWrapper,
   ) => (node: React.ReactNode) => ReactWrapper;
   slots?: Record<string, SlotTestingOptions>;
+  components?: Record<string, SlotTestingOptions>;
```

Comment:
leave a comment explaining the current usage of `components` and `slots`?
e.g., 
`slots` are for Material UI v6 and Joy UI
`components` are for Material UI v6; Expected to be removed from v7

Reply:
Good point.